zendesk/search_test.go

```diff
@@ -0,0 +1,26 @@
+package zendesk
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestSearchTickets(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test in short mode.")
+	}
```

Comment:
Do these tests take a long time?

Reply:
No, it was fast.. but all the test in this wrapper have that option so I added there